examples/dynamo/vgg16_fp8_ptq.py

```diff
@@ -226,11 +233,18 @@ def calibrate_loop(model):
     with export_torch_mode():
         # Compile the model with Torch-TensorRT Dynamo backend
         input_tensor = images.cuda()
-        exp_program = torch.export.export(model, (input_tensor,))
+        # torch.export.export() failed due to RuntimeError: Attempting to use FunctionalTensor on its own. Instead, please use it with a corresponding FunctionalTensorMode()
+        from torch.export._trace import _export
+
+        exp_program = _export(model, (input_tensor,))
+        if args.quantize_type == "int8":
```

Comment:
Shouldnt these be joined with a default set like `enabled_precisions = {torch.float, torch.half}`